release-pipeline.yml

```diff
@@ -7,8 +7,8 @@ parameters:
     type: string
     default: 'azcopy-'
 
-  - name: test_mode
-    displayName: 'Test Mode'
+  - name: publish_to_container
```

Comment:
Does this flag also prevent actual publishing to the Msft repos or only perform a dry run of publishing to msft repos?

Reply:
for publishing to **https://packages.microsoft.com/** repo we have a different flag named **publish_artifacts** 

**publish_to_container** this flag publishes to **azcopyvnextrelease** container if ticked else it publishes to **azcopyprivatedrops** container which is a test container.
**Test release**: https://ms.portal.azure.com/#view/Microsoft_Azure_Storage/ContainerMenuBlade/~/overview/storageAccountId/%2Fsubscriptions%2Fba45b233-e2ef-4169-8808-49eb0d8eba0d%2FresourceGroups%2FXClient%2Fproviders%2FMicrosoft.Storage%2FstorageAccounts%2Fazcopyprivatedrops/path/releases/etag/%220x8DB86D2AF30CD5B%22/defaultEncryptionScope/%24account-encryption-key/denyEncryptionScopeOverride~/false/defaultId//publicAccessVal/None